rmk-config/src/lib.rs

```diff
@@ -48,14 +47,16 @@ pub struct KeyboardTomlConfig {
     storage: Option<StorageConfig>,
     /// Ble config
     ble: Option<BleConfig>,
+    /// Chip-specific configs (e.g., [chip.nrf52840])
+    chip: Option<HashMap<String, ChipConfig>>,
```

Comment:
I'm thinking about making the supported chip-specific config an enum? Current implementation accepts **any** config in `keyboard.toml`, maybe we should make every chip-specific config a struct?